src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/help-containerCapStr.html

```diff
@@ -1,4 +1,4 @@
 <div>
     The maximum number of concurrently running agent pods that are permitted in this Kubernetes Cloud.
-    If set to empty it means no limit. Set to 0 means no pod will ever be started.
-</div>
+    If set to empty or 0 it means no limit.
```

Comment:
```suggestion
    If set to empty it means no limit.
```

No reason to confuse people by offering something to type in which would just be changed to blank on save.

Reply:
Agreed, thanks.